app/views/reports/regions/_block_tables.html.erb

```diff
@@ -0,0 +1,277 @@
+<h3 class="mt-48px mb-8px fs-28px c-black">
```

Comment:
This whole view could probably be consolidated w/ the facility_tables view, but not yet...